.drone.yml

```diff
@@ -17,6 +17,16 @@ pipeline:
       branch: master
       event: push
 
+  publish:
+    image: plugins/docker
+    repo: plugins/docker
```

Comment:
I don't think that brad want's to add this to the same image as this is generally feature-frozen.